kfs-core/src/main/resources/edu/arizona/kfs/fp/batch/pcdhDigesterRules.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0"?>
+<!DOCTYPE digester-rules 
+    PUBLIC "-//Jakarta Apache //DTD digester-rules XML V1.0//EN" 
+    "http://jakarta.apache.org/commons/digester/dtds/digester-rules.dtd">
+
+<digester-rules>
+    <pattern value="cardHolders">
+        <object-create-rule classname="java.util.ArrayList" />
+
+        <pattern value="cardHolder">
+            <object-create-rule classname="edu.arizona.kfs.fp.businessobject.ProcurementCardHolderLoad" />
+
+            <call-method-rule pattern="creditCardNumber" methodname="setCreditCardNumber" paramcount="0" />
```

Comment:
I don't think this is necessary because the last four can be calculated by a substring of the creditCardNumber